lib/saddle/requester.rb

```diff
@@ -2,7 +2,8 @@
 
 require 'faraday'
 require 'faraday_middleware'
-
+require 'saddle/faraday/request'
+require 'saddle/faraday/rack_builder'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.